README.md

```diff
@@ -1,6 +1,567 @@
-germanium
-=========
+# Germanium
 
 [![License](https://img.shields.io/badge/MIT-Clause-blue.svg)](https://opensource.org/licenses/MIT)
 
-Helpful methods for Python Selenium
+The main reason of the library was to create wrapper over selenium framework for Django unit tests, but over time the library was changed to test utils for the standard Django unit tests.
+
+## Installation
+
+- Install `django-germanium` with the `pip` command:
+
+```bash
+pip install django-germanium
+```
+
+Set the germanium test runner in your settings:
+
+```python
+TEST_RUNNER = 'germanium.django.runner.GermaniumDiscoverRunner'
+```
+
+## Configuration
+
+### Fixtures
+
+If you want to use fixtures for the all test cases which uses `GermaniumTestCase` test case you can define path to the file in your django settings with parameter:
+
+```python
+GERMANIUM_GLOBAL_FIXTURES = {
+    'default': [
+        os.path.join(PROJECT_DIR, 'data/common/test/init_data.json'),
+    ]
+}
+```
+The key `'default'` is name of the database which is configured in your django `DATABASES` setting.
+
+### Multiple DB testing
+
+Django rollbacks only primary database in `TestCase` by default. With `GermaniumTestCase` you can turn on the rollbacks on the all databases with setting:
+
+```python
+GERMANIUM_TEST_ALL_DATABASES = True
+```
+
+## Test Cases
+
+The library provides several test cases and mixins which simplify test definitions. There are in the package `test_cases` which is divided into modules:
+ * `germanium.tests_cases.default` - main Germanium test cases
+ * `germanium.tests_cases.auth` - contains a test case class which can help with user authentication
+ * `germanium.tests_cases.client` - test cases with methods helping with testing endpoints and web pages
+ * `germanium.tests_cases.rest` - test cases with methods helping with testing REST API
+ * `germanium.tests_cases.models` - depricated tests case helping with model tests
+ * `germanium.tests_cases.migrations` - a migration tests case helper
+
+### Default
+
+Default module contains four classes with this inheritance:
+```python
+                                       django.test.testcases.TestCase        \
+GermaniumSimpleTestCaseMixin         - GermaniumTestCaseMixin                - GermaniumTestCase
+                                     \ GermaniumSimpleTestCase
+django.test.testcases.SimpleTestCase /  
+```
```

Comment:
This code block looks strange. Does it render nicely in the html docs? You can use images if you want to show a diagram.